main/src/ca/uwaterloo/flix/language/phase/Typer.scala

```diff
@@ -227,7 +227,7 @@ object Typer {
           ///
           val initialSubst = getSubstFromParams(fparams0)
 
-          run(initialSubst, Map.empty) match { // TODO renv
+          run(initialSubst, Rigidity.emptyEnv) match { // TODO renv
```

Comment:
What about all these TODOs?

Reply:
The ones in the Typer are left for the followup PR.